src/participant_network.star

```diff
@@ -187,8 +187,10 @@ def launch_participant_network(plan, participants, network_params, global_log_le
 		index_str = zfill_custom(index+1, zfill_calculator(participants))
 
 		cl_service_name = "cl-{0}-{1}-{2}".format(index_str, cl_client_type, el_client_type)
-
-		new_cl_node_validator_keystores = preregistered_validator_keys_for_nodes[index]
+		if participant.validator_count != 0:
```

Comment:
lets just initialize new_cl_node_validator_keystores to None before the if condition, that wauy we don't need the if condition